modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/shared/util/duration.es.js

```diff
@@ -72,3 +72,46 @@ export function getDurationValues(durationValue) {
 		seconds: fullDuration.seconds() || null,
 	};
 }
+
```

Comment:
Hey @jotascript, it would be nice if we had tests for this function. What do you think?

Thx

Reply:
@rafaprax This function is already covered by the test of InstanceListPageItem.es.js.

Reply:
@jotascript I mean unit tests for each possible usages.